getting_started.Rmd

```diff
@@ -97,9 +97,9 @@ If you have organization admin privileges and plan on making multiple courses wi
 
 - Click `New repository secret`/`New organization secret` button.
 - Under `Name` you must use `GH_PAT`.
-- For `Secret`: Create a personal access token [following these instructions](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token). 
+- For `Secret`: Create a personal access token [following these instructions](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token).
```

Comment:
```suggestion
- For `Secret`: Create a personal access token [following these instructions](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token#creating-a-token) to create a "Classic" token.
```

Reply:
now that there is another type... for the next line to make sense we need to add this

Reply:
btw we only had people do repo for ottr web ... does it need to be workflow too for that?

Reply:
I think (but am not 100% sure that only repo is needed). We should verify this.